workers/python/Dockerfile

```diff
@@ -69,8 +69,7 @@ WORKDIR /app
 
 COPY workers/python/python/ .
 
-# ENV GLOBAL_WORKING_DIR_ENABLED true
-# ENV GLOBAL_WORKING_DIR /tmp/unstructured
-# ENV HUGGINGFACE_HUB_CACHE /tmp/huggingface
+# Prepare worker resources.
+RUN python3.11 init_resources.py
```

Comment:
> How long does this take and how big is the download? Can we extract it before the COPY workers/python/python/ . so that every code change does not trigger the model download?

Will do

> Is it a good idea to download the resource in the Dockerfile? They may not be needed for unstructured backend so they can be downloaded lazily. I saw the https://github.com/i-am-bee/bee-api/issues/49#issuecomment-2464890585, just saying that we might save some amount of data downloaded by bee-stack users

We shouldn't depend on third party server nor make the bootstrap process of pods longer than necessary. We can eventually separate images of unstructured and docling. Bee-stack users can opt out of python workers.